examples/step-58/doc/results.dox

```diff
@@ -0,0 +1,207 @@
+if you look at the shaded part of the domain outside the circle of
+radius 0.7 in which the potential is zero -- you can see how everytime
+one of the bumps (showing the amplitude $|\psi_h(\mathbf x,t)|^2$)
+bumps into the area where the potential is large: a wave travels
+outbound from there. Take a look at the video:
+
+@htmlonly
+<p align="center">
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/nraszP3GZHk"
+   frameborder="0"
+   allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+   allowfullscreen></iframe>
+ </p>
+@endhtmlonly
+
+So why did I end up shading the area where the potential $V(\mathbf x)$ is
+large? In that outside region, the solution is relatively small. It is also
+relatively smooth. As a consequence, to some approximate degree, the
+equation in that region simplifies to
+@f[
+  - i \frac{\partial \psi}{\partial t}
+  + V \psi
+  \approx 0,
+@f]
+or maybe easier to read:
+@f[
+  \frac{\partial \psi}{\partial t}
+  \approx - i V \psi.
+@f]
+To the degree to which this approximation is valid (which, among other things,
+eliminates the traveling waves you can see in the video), this equation has
+a solution
+@f[
+  \psi(\mathbf x, t) = \psi(\mathbf x, 0) e^{-i V t}.
+@f]
+Because $V$ is large, this means that the phase *rotates quite rapidly*.
+If you focus on the semi-transparent outer part of the domain, you can
+see that. If one colors this region in the same way as the inner part of
+the domain, this rapidly flashing outer part may be psychedelic, but is also
+distracting of what's happening on the inside; it's also quite hard to
+actually see the radiating waves that are easy to see at the beginning
+of the video.
+
+
+<a name="extensions"></a>
+<h3>Possibilities for extensions</h3>
+
+<h4> Better linear solvers </h4>
+
+The solver chosen here is just too simple. It is also not efficient.
+What we do is just punt the matrix to a sparse direct solver in every
+time step and let it find the solution of the linear system. But we
+know that we could do far better:
+
+- First, we should make use of the fact that the matrix doesn't
+  actually change from time step to time step. This is an artifact
+  of the fact that we here have constant boundary values and that
+  we don't change the time step size -- two assumptions that might
+  not be true in actual applications. But at least in cases where this
+  does happen to be the case, it would make sense to only factorize
+  the matrix once (i.e., compute $L$ and $U$ factors once) and then
+  use these factors for all following time steps until the matrix
+  $C$ changes and requires a new factorization. The interface of the
+  SparseDirectUMFPACK class allows for this.
+
+- Ultimately, however, sparse direct solvers are only efficient for
+  relatively small problems, say up to a few 100,000 unknowns. Beyond
+  this, one needs iterative solvers such as Conjugate Gradients (for
```

Comment:
`Conjugate Gradients` -> `Conjugate Gradient` / `the family of Conjugate Gradient approaches`

I believe that you are implying that CG/CGS/FlexCG are suitable, but I wonder if the link between `Conjugate Gradient` and the canonical CG method is just too strong for the plural form that you chose to be easily understood.